presto-docs/README.md

```diff
@@ -114,6 +122,23 @@ make clean livehtml
 From now on the docs are available at
 [http://localhost:8000](http://localhost:8000).
 
+## Versioning
+
+The version displayed in the resulting HTML is read from the top level Maven
+`pom.xml` file `version` field, by default.
+
+To deploy a specific documentation set (e.g. a SNAPSHOT version) as release
+version you have to override the pom version with the `PRESTO_VERSION`
+environment variable.
+
+```bash
+export PRESTO_VERSION
```

Comment:
I think it would be better to write this inline with a version specified:
```
PRESTO_VERSION=1234 make clean html

Reply:
@mosabua I'll merge this now, but please send a PR for this if you agree

Reply:
https://github.com/prestosql/presto/pull/2244